acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/multiplatformsigner/AzureSignerMultiplatformSigning.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests.multiplatformsigner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.fail;
+
+import tech.pegasys.ethsigner.tests.dsl.node.NodeConfiguration;
+import tech.pegasys.ethsigner.tests.dsl.node.NodeConfigurationBuilder;
+import tech.pegasys.ethsigner.tests.dsl.node.NodePorts;
+import tech.pegasys.ethsigner.tests.dsl.signer.Signer;
+import tech.pegasys.ethsigner.tests.dsl.signer.SignerConfiguration;
+import tech.pegasys.ethsigner.tests.dsl.signer.SignerConfigurationBuilder;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+public class AzureSignerMultiplatformSigning {
+
+  private static final String clientId = System.getenv("ETHSIGNER_AZURE_CLIENT_ID");
+  private static final String clientSecret = System.getenv("ETHSIGNER_AZURE_CLIENT_SECRET");
+
+  final String AZURE_ETHEREUM_ADDRESS = "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73";
+
+  private static Signer ethSigner;
+
+  @TempDir static Path tomlDirectory;
+
+  @BeforeAll
+  public static void preChecks() {
+    Assumptions.assumeTrue(
+        clientId != null && clientSecret != null,
+        "Ensure Azure client id and client secret env variables are set");
```

Comment:
aren't these specified in the TOML files?

Reply:
Yes they typically are - however, given these 2 values give you access to our signing capability in Azure, we do not want to expose them in our repo (thus can never check these strings into git). Instead, we put them in env vars (which can be injected by circle CI), then copy them into place - thus the Azure TOML file must be "hand-created" in this test.